src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/PackageSourceMappingUtility.cs

```diff
@@ -91,5 +91,52 @@ internal static class PackageSourceMappingUtility
 
             mappingProvider.SavePackageSourceMappings(newAndExistingPackageSourceMappingItems);
         }
+
+        internal static string? GetNewSourceMappingSourceName(PackageSourceMapping packageSourceMapping, PackageSourceMoniker activePackageSourceMoniker)
+        {
+            string? sourceMappingSourceName = packageSourceMapping.IsEnabled
+                && !activePackageSourceMoniker.IsAggregateSource
+                ? activePackageSourceMoniker.PackageSourceNames.First() : null;
+
+            return sourceMappingSourceName;
+        }
+
+        internal static void GetNewSourceMappingsFromAddedPackages(ref Dictionary<string, SortedSet<string>>? newSourceMappings, string newMappingSourceName, List<AccessiblePackageIdentity> added, PackageSourceMapping packageSourceMapping)
+        {
+            if (newMappingSourceName is null || added.Count == 0 || packageSourceMapping is null)
+            {
+                return;
+            }
+
+            List<string> addedPackagesWithNoSourceMappings = added.Select(_ => _.Id)
```

Comment:
Good callout! I've been trying to use throw-aways more often, when I don't care about the variable name and don't really need one. To me, arbitrary letters are unnecessary noise. (`i` probably isn't what I'd choose, but maybe `p` for "package" or the word itself?)

Reply:
It did look a bit unusual at first, but ultimately I find the code readable.

Reply:
Thanks for explaining that. I realize now I started this habit to avoid redundant naming, and it's actually just been an underscore variable name all along 😁.